BulgarskiChiflikGuestHouse.Web/Views/Home/Index.cshtml

```diff
@@ -118,16 +118,16 @@
 					<div class="col-lg-6">
 						<div class="row g-3 mt-2">
 							<div class="col-6 text-end">
-								<img class="img-fluid rounded w-75 wow zoomIn" data-wow-delay="0.1s" src="img/main2.jpg">
+								<img class="img-fluid rounded w-75 mt-5 wow zoomIn" data-wow-delay="0.1s" src="img/GuestHousePoolView1.jpg">
 							</div>
 							<div class="col-6 text-start">
-								<img class="img-fluid rounded w-100 wow zoomIn" data-wow-delay="0.3s" src="img/main3.jpg">
+								<img class="img-fluid rounded w-100 wow zoomIn" data-wow-delay="0.3s" src="img/GuestHouseRearSideView.jpg">
 							</div>
 							<div class="col-6 text-end">
-								<img class="img-fluid rounded w-50 wow zoomIn" data-wow-delay="0.5s" src="img/main1.jpg">
+								<img class="img-fluid rounded w-50 wow zoomIn" data-wow-delay="0.5s" src="img/GuestHouseFontalView1.jpg">
```

Comment:
Do you have typo here GuestHouseFontalView1 => GuestHouseFrontalView1?

Reply:
Fixed it.